### PR TITLE
Fix filelist overlap with header ref #16076

### PR DESCRIPTION
**Pull request by GretaD**
ref #16076
Before
before1
After
after1

### CODE REVIEW

File: core/css/styles.scss [hunk 1]
```diff
@@ -50,6 +50,7 @@ caption, th, td {
 
 table, td, th {
 	vertical-align: middle;
+	padding-top: 40px;
```
Comment (skjnldsv) [inline core/css/styles.scss]: This seems like a far too wide css selector.
That means every table, td and th will have a padding. Yo uneed to find the css rule that only fit the filelist header :)

**Review by skjnldsv — CHANGES_REQUESTED** [review]

File: core/css/styles.scss (hunk 1 above)
Comment (GretaD) [reply core/css/styles.scss]: oh thats the general css file, thats a stupid suggestion then. Will push the right one asap

Comment (skjnldsv) [reply core/css/styles.scss]: Nothing stupid here :)

File: apps/files/css/files.scss [hunk 2]
```diff
@@ -70,7 +70,8 @@
 			position: -webkit-sticky;
 			position: sticky;
 			// header + breadcrumbs
-			top: $header-height + 44px;
+			top: $header-height + 2px;
```
Comment (georgehrke) [inline apps/files/css/files.scss]: How did you come up with the 2px here? :)

Comment (GretaD) [reply apps/files/css/files.scss]: > How did you come up with the 2px here? :)

No matter what i do, it doesnt work if I dont change this. What do you propose? Am i editing the wrong file?

Comment (juliushaertl) [reply apps/files/css/files.scss]: Just removing the 2px worked fine for me locally

Comment (GretaD) [reply apps/files/css/files.scss]: My first file is a pdf file and the next one is an image, the pdf file has borders, the image doesnt. For some reasons it looked better with 2px, but actually it doesnt now that i see carefully :) so im removing the 2px. Thanks @juliushaertl

**Review by juliusknorr — APPROVED** [review]

File: apps/files/css/files.scss (hunk 2 above)
Comment (juliushaertl) [reply apps/files/css/files.scss]: You're welcome. The border around the icon is already filed as an issue in #14849

**Review by georgehrke — APPROVED** [review]
👍

**Pull request merged**